sites/frontend/components/Header/Nav/SubNav.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import { styled } from '@guardian/guui';
+import type { LinkType } from './__config__';
+
+type Props = {
+    parent?: LinkType,
+    links: Array<LinkType>,
+};
+
+const UlStyled = styled('ul')({
+    listStyle: 'none',
+    li: {
+        float: 'left',
+    },
+    a: {
+        display: 'block',
+        fontWeight: 400,
+        color: '#121212',
+        padding: '0 0.3125rem',
+        fontSize: '1rem',
+        height: '2.625rem',
+        lineHeight: '2.625rem',
```

Comment:
lots of magic numbers here

Reply:
🌠 yeah - I think all of the styling could be improved. But I'm tempted to do that in the follow-up refactoring depending on if people are okay with that.

Reply:
I'm guessing you pulled these `rem` numbers from the dev tools, we'll want to convert these to `px` units as eventually we'll be applying a px-to-rem conversion at some point. I'm happy for this to be done in a later refactor.